bindings/python/libmata.pxd

```diff
@@ -175,51 +175,51 @@ cdef extern from "mata/nfa.hh" namespace "Mata::Nfa":
         void clear_final()
         void unify_initial()
         void unify_final()
-        SymbolSet get_symbols()
+        SymbolSet get_used_symbols()
         void add_trans(CTrans) except +
         void add_trans(State, Symbol, State) except +
         void remove_trans(CTrans) except +
         void remove_trans(State, Symbol, State) except +
         bool has_trans(CTrans)
         bool has_trans(State, Symbol, State)
-        bool trans_empty()
+        bool has_no_transitions()
         bool nothing_in_trans()
         bool is_state(State)
         size_t get_num_of_trans()
         StateSet post(StateSet&, Symbol)
         CNfa.const_iterator begin()
         CNfa.const_iterator end()
-        size_t get_num_of_states()
+        size_t size()
         void increase_size(size_t)
         void increase_size_for_state(State)
-        State add_new_state()
+        State add_state()
         void print_to_DOT(ostream)
         WordSet get_shortest_words()
-        TransitionList get_transitions_from(State)
+        Moves get_moves_from(State)
         vector[CTrans] get_transitions_to(State)
         vector[CTrans] get_trans_as_sequence()
         vector[CTrans] get_trans_from_as_sequence(State)
         void trim()
-        void get_digraph(CNfa&)
+        void get_one_letter_nfa(CNfa&)
```

Comment:
maybe get_one_letter_aut ?

Reply:
can be, we don't have any other FA anyway.